internal/commands/points/points.go

```diff
@@ -0,0 +1,82 @@
+package points
+
+import (
+	"fmt"
+	"net/http"
+	"encoding/json"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/pajbot/basecommand"
+	"github.com/pajbot/pajbot2-discord/internal/config"
+	"github.com/pajbot/pajbot2-discord/pkg"
+	"github.com/pajbot/pajbot2-discord/pkg/commands"
+	"github.com/pajbot/pajbot2-discord/pkg/utils"
+)
+
+func init() {
+	commands.Register([]string{"$points"}, New())
+}
+
+var _ pkg.Command = &Command{}
+
+type Command struct {
+	basecommand.Command
+}
+
+type Points struct {
+	Id   int64  `json:"id"`
+	Name string `json:"username"`
+	NameRaw string `json:"username_raw"`
```

Comment:
Change NameRaw to `DisplayName` - it's a more accurate naming than my old "username_raw"